projects/saffron/index.js

```diff
@@ -0,0 +1,28 @@
+const abi = require("./abi.json");
+const { addFundsInMasterChef } = require("../helper/masterchef");
+
+const SaffronStakingV2Contract = "0x4eB4C5911e931667fE1647428F38401aB1661763";
+
+const ethTvl = async (chainBlocks) => {
+  const balances = {};
+
+  await addFundsInMasterChef(
```

Comment:
shouldn't we count the tvl on tranches for saffron? tvl on that masterchef contract is mostly SFI and SFI pool2 staked, which would be better to separate into staking/pool2

Reply:
yeah, should we. But I thougth as there is a pool PRT/ETH, it's better putting all together through masterchef, maybe will be created more pools like this, and masterchef should be better.